src/targets/gpu/fuse_ops.cpp

```diff
@@ -893,11 +898,13 @@ void fuse_ops::apply(module& m) const
     match::find_matches(m, find_conv_pointwise{ctx}, find_conv_bias_relu{ctx}, find_conv_bias{ctx});
     run_passes(m, {dead_code_elimination{}});
     match::find_matches(m,
-                        find_gemm_pointwise{},
                         find_layernorm_pointwise{},
                         find_concat_pointwise{},
                         find_contiguous_tranpose_gemm{},
                         find_commutative_broadcast{});
+    #ifdef MIGRAPHX_USE_ROCBLAS
+        match::find_matches(m, find_gemm_pointwise{});
```

Comment:
This should not be moved here, it should stay in the list where it was at if rocblas is enabled.